design/resources.go

```diff
@@ -146,6 +146,27 @@ var _ = Resource("workitemtype", func() {
 	})
 })
 
+var _ = Resource("user", func() {
+	BasePath("/user")
```

Comment:
The naming currently mimics the style from GH; https://developer.github.com/v3/users/#get-the-authenticated-user